### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by truetechcode**

### CODE REVIEW

File: src/javascript/weatherModule.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    .then((response) => response.json())
+    .then((response) => {
+      if (response) {
+        const {
+          weather, tempF, tempC, icon, isDay,
+        } = response.response.ob;
+        tempArray = [`${tempF}&#8457;`, `${tempC}&#8451;`];
+        const { lat, long } = response.response.loc;
+        const content = `<p class="header">
+        Location: <span>${ci}, ${co}</span>
```
Comment (rvvergara) [inline src/javascript/weatherModule.js]: I have another problem with this function. There are lots of `html` code in here. We aren't supposed to generate our html using JavaScript. It's bad practice as it hampers performance. The browser will wait until ALL JS script are loaded before showing these html elements we are dynamically generating here. The only thing that our JS should be doing as far as the DOM is concerned is attaching to **specific** dom elements (which we are querying thru `document.getElementByID`, etc) the data we are dynamically generating.

File: src/javascript/weatherModule.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-unused-vars */
+const container = document.querySelector('div[weather] .content');
+const location = document.createElement('div');
+const img = document.createElement('img');
+location.classList.add('location');
+let tempArray = [];
```
Comment (rvvergara) [inline src/javascript/weatherModule.js]: Let us avoid using global variables that hold data. The only global variables we should be having are DOM elements like the ones above.

File: src/javascript/weatherModule.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-unused-vars */
```
Comment (rvvergara) [inline src/javascript/weatherModule.js]: It is not a recommended practice to disable eslint rules inline like this. We do have the `eslintrc` file to include rules. Another thing is that why are we disabling this `no-unused-var` rule? It's intended for us to know when we are declaring a variable and not using it (useless variable in simple terms).

File: src/javascript/weatherModule.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      tempHolder.innerHTML = tempF;
+    } else {
+      tempHolder.innerHTML = tempC;
+    }
+    tempCount += 1;
+    e.preventDefault();
+  });
+};
+
+const loadWeather = (ci, co) => {
```
[the remark this answers is not in the document]
Comment (truetechcode) [reply src/javascript/weatherModule.js]: The City in question actually exist in the US, it is for cases like this that I have left the feature in place. Thanks

**Pull request merged**